### PR TITLE
Ronan f/#4 update fn signature

### LINKED ISSUE #4: Refactor assertMessageForDate
## Background

When we need to pass more than 1 input to a function, consider passing an object (referred to as "named arguments" in Ember). A hash (POJO) helps remove order dependency and clarify the invocation site.

## Problem

The file `test/cron/daily-message.js` has a custom assertion called `assertMessageForDate`. Currently, it has 4 "positional arguments."

## TODOs

- [ ] Update the signature of `assertMessageForDate`.

    ```javascript
    // Before
    async function assertMessageForDate(today, releaseDate, message, released = []) { ... }

    // After
    async function assertMessageForDate({ todayDate, releaseDate, message, released = [] }) { ... }
    ```

- [ ] Update the invocations in `test/cron/daily-message.js`. For example,

   ```javascript
    // Before
    await assertMessageForDate('2020-07-13', '2020-07-20', 'Release week starts next week on 2020-07-20 are we all prepared? :lts:');

    // After
    await assertMessageForDate({
      todayDate: '2020-07-13',
      releaseDate: '2020-07-20',
      message: 'Release week starts next week on 2020-07-20 are we all prepared? :lts:',
    });
    ```

### PR DESCRIPTION
**Pull request by ronan-f**
- [x] Update signature for `assertMessageForDate`
- [x] Update calls to `assertMessageForDate`
- [x] Tests passing

I believe this is what the issue required. Let me know if any changes are needed.

### CODE REVIEW

File: test/cron/daily-message.js [hunk 1]
```diff
@@ -33,47 +38,70 @@ describe('daily-message cron job', function () {
...
   });
 
   it('should send a message on the first day of release week', async function () {
-    await assertMessageForDate('2020-07-20', '2020-07-20', `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!
-
-If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`);
+    await assertMessageForDate({
+      todayDate: '2020-07-20',
+      releaseDate: '2020-07-20',
+      message: `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something! If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`,
```
Comment (ijlee2) [inline test/cron/daily-message.js]: I think the newline in line 41 (deleted) might have been deliberately added. To keep the behavior the same, but to format the code better, what do you think about this approach?

```suggestion
      message: [
        `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!`,
        '',
        `If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`
      ].join('\n'),
```

**Review by ijlee2 — COMMENTED** [review]
The changes look great! I left 1 suggestion that I think we'll need to implement. Can you have a look?

File: test/cron/daily-message.js (hunk 1 above)
Comment (ronan-f) [reply test/cron/daily-message.js]: Ah I see. I should have clarified that. I like your approach with `.join()`. I'll update now.

**Review by ijlee2 — APPROVED** [review]
🎉

**Pull request merged**